doc/changelog/06-ssreflect/10022-ssr-under-setoid.rst

```diff
@@ -0,0 +1,21 @@
+- Generalize tactics :tacn:`under` and :tacn:`over` for any registered
+  setoid equality. More precisely, assume the given context lemma has
+  type `forall f1 f2, .. -> (forall i, R1 (f1 i) (f2 i)) -> R2 f1 f2`.
+  The first step performed by :tacn:`under` (since Coq 8.10) amounts
+  to calling the tactic :tacn:`rewrite <rewrite (ssreflect)>`, which
+  itself relies on :tacn:`setoid_rewrite` if need be. So this step was
+  already compatible with a double implication or setoid equality for
+  the conclusion head symbol `R2`. But a further step consists in
+  massaging the generated subgoal `R1 (f1 i) (?f2 i)` to protect it
```

Comment:
Instead of massage I'd use "recognize"

Reply:
@gares Indeed, my wording here was not very nice;
but it seems to me that `recognize` says less than what is done here?
so would you be OK with **`processing`** or **`wrapping`**?
(or do you have a strong preference for `recognizing`)

Reply:
no, it is just that massaging seemed very informal. Maybe "tag"? Wrapping also works for me

Reply:
OK thanks @gares, so let's go for "tagging" then :) (I just force-pushed to amend that commit.)